### PR TITLE
(chore) change ‘vacancy’ to ‘job’ in UI

### PR DESCRIPTION
**Pull request by gazaston**

### CODE REVIEW

**Comment by gazaston** [comment]
[Change "Vacancies" to "Jobs" in the UI, site-wide](https://trello.com/c/8k3PD8NH/143-change-vacancies-to-jobs-in-the-ui-site-wide)

File: config/locales/en.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ en:
     flexible_working_info: 'Yes. Email %{mailto} for more information.'
     starts_on: 'Expected start date'
     ends_on: 'End date'
-    expired: 'This vacancy has expired'
+    expired: 'This job has expired'
```
Comment (despo) [inline config/locales/en.yml]: @gazaston  This is where I got stuck before - should this not be `This job vacancy has expired?` It does not read correctly as it is.

Comment (gazaston) [reply config/locales/en.yml]: Perhaps 'This job posting has expired' instead?

Comment (despo) [reply config/locales/en.yml]: Actually `job post` as `job vacancy` is even more confusing.

**Pull request merged**